### PR TITLE
properly route users to RIDP verification page

### PR DESCRIPTION
**Pull request by harshaellanki**
ME-180460028

### CODE REVIEW

File: app/assets/javascripts/consumer_role.js [hunk 1]
```diff
@@ -11,8 +11,13 @@ $(document).on('change', "#person_no_dc_address, #dependent_no_dc_address, #no_d
 });
 
 $(document).on('ready', function(){
+    if(`EnrollRegistry.feature_enabled?(:ridp_h139) && Rails.env.production? && !(ENV['ENROLL_REVIEW_ENVIRONMENT'] == 'true')`) {
```
Comment (markgoho) [inline app/assets/javascripts/consumer_role.js:14]: how is JavaScript able to evaluate this logic?

File: app/assets/javascripts/consumer_role.js [hunk 2]
```diff
@@ -11,8 +11,13 @@ $(document).on('change', "#person_no_dc_address, #dependent_no_dc_address, #no_d
...
+      var path = "/insured/fdsh_ridp_verifications/new";
```
Comment (markgoho) [inline app/assets/javascripts/consumer_role.js:15]: declare the variable outside the `if` block, then reassign within the conditional

``` js
let path;
if (true) {
  path = '';
} else {
  path = '';
}
```

**Review by markgoho — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/consumer_role.js [hunk 3]
```diff
@@ -11,8 +11,13 @@ $(document).on('change', "#person_no_dc_address, #dependent_no_dc_address, #no_d
 });
 
 $(document).on('ready', function(){
```
Comment (ZASMan) [inline app/assets/javascripts/consumer_role.js:13]: How is this gonna render if its just JS

File: app/assets/javascripts/consumer_role.js (hunk 1 above)
Comment (harshaellanki) [reply app/assets/javascripts/consumer_role.js:14]: i added it in escape strings

File: app/assets/javascripts/consumer_role.js (hunk 3 above)
Comment (harshaellanki) [reply app/assets/javascripts/consumer_role.js:13]: @ZASMan it is document onclick

File: app/assets/javascripts/consumer_role.js (hunk 1 above)
Comment (markgoho) [reply app/assets/javascripts/consumer_role.js:14]: the backticks are used for string interpolation or tagged template literals -- they have no affect on how code is evaluated

**Review by markgoho — APPROVED** [review]
moved JS file to erb file so that we could gain access to server-side values

this change fixes a condition where users would not be able to get to the verification page after choosing Disagree, then choosing Agree

**Pull request merged**